cmd/deploy/deploy.go

```diff
@@ -43,6 +44,15 @@ func NewTeardown() *cobra.Command {
 	return teardownCmd
 }
 
+func NewApply() *cobra.Command {
```

Comment:
I don't mind adding the apply command if we also add testing for `applyFn`.  I would like to see our test coverage generally increase as we add new code.

Reply:
Decided to delete the command, we can decide at a later date how much of kubectl we want to wrap in KNE cli if any at all. Currently I would consider it a necessary tool to use along side the kne cli